pkgs/applications/gis/grass/default.nix

```diff
@@ -15,6 +15,10 @@ stdenv.mkDerivation {
   readline ffmpeg makeWrapper wxGTK30 netcdf geos postgresql mysql.connector-c blas ]
     ++ (with python2Packages; [ python dateutil wxPython30 numpy ]);
 
+  # On Darwin the installer tries to symlink the help files into a system
+  # directory
+  patches = [] ++ lib.optional stdenv.isDarwin [ ./no_symbolic_links.patch ];
```

Comment:
Does the patch actually need to be darwin-specific? By that I mean that the functionality it patches only changes on Darwin, but it doesn't hurt other platforms, right?

The reason I ask is that it makes it more likely for Linux folks to notice shit like this breaking if it affects their builds. Imagine someone a few months from now upgrades the package and the patch breaks. The linux person updating the package won't notice and it'll break again on Darwin. If instead you make the patch unconditional, it'll break immediately and someone will know to fix it.

In general, I'd try to make things as unconditional as possible. If we were trying to be careful about downstream hash changes and such, I'd consider that, but I don't think that's true here, right?

Reply:
The patch only affects codepaths hit by Darwin (as otherwise the linux build would fail) so I can make it unconditional. Is there anything else that needs changing?

Reply:
I don't have any other comments.

Reply:
Nope, I think it generally looks good, thanks 👍